test/lib/ansible_test/_internal/coverage_util.py

```diff
@@ -254,13 +255,13 @@ def generate_ansible_coverage_config() -> str:
 parallel = True
 
 omit =
-    */python*/dist-packages/*
-    */python*/site-packages/*
-    */python*/distutils/*
-    */pyshared/*
-    */pytest
-    */AnsiballZ_*.py
-    */test/results/*
+    **/python*/dist-packages/**
```

Comment:
@mattclay should we revert these after all?

Reply:
Just to confirm, the new `**` behavior is only for matching in the middle of a path? The `*/` and `/*` matches at the start and end of a path continue to work without `**`?

If that's the case, then yes, we should revert the path changes, since it seems the coverage docs will be updated, rather than the implementation.

Reply:
Yes, that's correct. I've been thinking of documenting this upstream, since it's unlikely that Ned will accept the original change I offered.